tests/full-disk-encryption.spec.ts

```diff
@@ -27,29 +25,18 @@ test.describe('The main page', () => {
             await storagePage.useEncryption();
 
             const passwordPopup = new EncryptionPasswordPopup(page);
-            await passwordPopup.fillPassword('nots3cr3t');
-            await passwordPopup.fillPasswordConfirmation('nots3cr3t');
+            await passwordPopup.fillPassword(UserActor.user.password);
```

Comment:
Just a minor comments, maybe here a potential risk, just think if we set different password for encryption and user password... Then maybe later PR we can set the interface for storage, and set its own password.

Reply:
We could add new properties to the interface if needs in the future or create a new one that extends the current one. But at the moment it doesn't seem to be useful. Do you prefer to add now these new properties to manage root password and encryption password?

Reply:
I agree with you, we can add that in the future, and now leave it like this is ok for me.